automation/azure-pipelines-unit.yml

```diff
@@ -64,15 +62,21 @@ steps:
       move $(license_file.secureFilePath) ./src/ -Force
 
 - task: PowerShell@2
+  displayName: Build code
   inputs:
     filePath: './src/build.ps1'
     arguments: '-Target 002-Build'
     workingDirectory: './src/'
-  displayName: Build
 
 - task: PowerShell@2
+  displayName: Run tests
```

Comment:
Please specify that this is Unit tests, since we'll have UI and API tests as well

Reply:
Changed name of task to "Run Unit tests". It doesn't added first time because yaml and pipeline name already had "unit" in name.